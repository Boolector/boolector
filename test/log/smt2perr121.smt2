(assert (let ((
; end-of-file comes ...
