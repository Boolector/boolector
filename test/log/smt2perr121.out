log/smt2perr121.smt2:3:1: expected symbol to be bound after '(' at line 1 column 15 but reached end-of-file
